ydb/library/yql/sql/v1/node.h

```diff
@@ -688,16 +688,23 @@ namespace NSQLTranslationV1 {
     };
 
     struct TColumnSchema {
+        enum class ETypeOfChange {
+            Nothing,
+            SetNullConstraint,
+            SetFamaly
```

Comment:
set family ?

Reply:
In set family query calls this constructor